tests/garage/tf/samplers/test_off_policy_vectorized_sampler.py

```diff
@@ -0,0 +1,68 @@
+import gym
+import tensorflow as tf
+
+from garage.experiment import LocalRunner
+from garage.np.exploration_strategies import OUStrategy
+from garage.replay_buffer import SimpleReplayBuffer
+from garage.tf.algos import DDPG
+from garage.tf.envs import TfEnv
+from garage.tf.policies import ContinuousMLPPolicy
+from garage.tf.q_functions import ContinuousMLPQFunction
+from garage.tf.samplers import OffPolicyVectorizedSampler
+from tests.fixtures import TfGraphTestCase
+
+
+class TestOffPolicyVectorizedSampler(TfGraphTestCase):
```

Comment:
can you test this without running an entire algorithm? it seems like you should be able to use a couple test fixtures and/or mocking to write this as a unit test.

Reply:
The sampler needs an algorithm policy to do sampling, but no algorithm is actually run, just to setup sampler.

Reply:
perhaps you can get by with a mock algorithms or policy? anyway, it's not a big deal.